### PR TITLE
PR4_3changes_longdivision

### PR DESCRIPTION
**Pull request by sheppardw**

### CODE REVIEW

**Comment by StevenClontz** [comment]
@sheppardw pull my latest commit into your codespace branch to get the fix for your long division (which is basically all the stuff @siwelwerd suggested, thanks Drew).

**Comment by sheppardw** [comment]
I sent a draft version here for PR-4Sec3. The long division just needs some minor changes. 
Requests:
-I need the code to make a blank square. I found a couple of examples for PreText but am not sure which one to use or how to incorporate this into my code. Please advise. :-)
-I could not get the divisor to appear properly without putting parentheses around it, but I am sure there is a way to do this. Please advise. :-)
-I need to find the code for synthetic division, which would mainly be a vertical line and a horizontal line, but I want to include the square blanks. I will keep looking but if you know of an easy way to incorporate this into the code, please advise. :-)

**Comment by siwelwerd** [comment]
> I need to find the code for synthetic division, which would mainly be a vertical line and a horizontal line

This should look a whole lot like the long division, but just swapping out a `)` for a `|`.

>I need the code to make a blank square

I feel like we have a hack to put squares around things in the Linear Algebra book (we put squares around pivots in matrices).  So that can likely be copied.  Another option is to use a `?` instead of a blank square, which is maybe more consistent with what we have done elsewhere.

**Comment by sheppardw** [comment]
Ok, the ? is easy to do! 
For the synthetic, I do not know how to reverse the vertical line to be above the horizontal instead of below though. I will look at it and see. If you know how to do this already, please let me know. Thank you!!

**Comment by sheppardw** [comment]
To move the line below, instead of "raise" type "lower" ? Just guessing.

**Comment by sheppardw** [comment]
Also how do I make the entire long division problem appear larger? I tried changing some numbers in the code but could not figure out what number to change for the size. Thanks!

**Comment by sheppardw** [comment]
nevermind on the raising and lowering...use the negative... got it!

**Comment by sheppardw** [comment]
Figured out the sizing issue too!

**Comment by StevenClontz** [comment]
Sounds good - either tag me back in if you want me to tweak anything, or mark this as ready for review if you think you've got it figured out. 😄

**Comment by sheppardw** [comment]
I still need to figure out spacing in the array because I was having trouble with that and I feel like the synthetic division, the vertical line, can look better. So I need help with those 2 things please. Other than that, this is ready for review. So I will wait to fix these issues first.

**Comment by sheppardw** [comment]
I can...If you look at the current code, and just bring the vertical line down to touch the horizontal at a clean corner, that is it!

**Comment by StevenClontz** [comment]
> I can...If you look at the current code, and just bring the vertical line down to touch the horizontal at a clean corner, that is it!

Gotcha - I'll take a look.

**Comment by StevenClontz** [comment]
Actually I'm still not clear what you're asking. This is what I see:

image

Can you draw what you're looking for?

File: source/04-PR/03.ptx [hunk 1]
```diff
@@ -345,75 +230,78 @@
...
+                              &amp; &amp; &amp; &amp; &amp; 3 &amp; &amp; 2 &amp; &amp;5 &amp; &amp; &amp; &amp;\\ 
+                              &amp; &amp; &amp; &amp; &amp; &amp; &amp; &amp; &amp; &amp; &amp;\\ 
+        
+        
+                              \end{array}
+                            \)
+                          </latex-image>
+                          </image>
+                        </statement>
+                    </task>
```
Comment (StevenClontz) [inline source/04-PR/03.ptx:294]:
```suggestion
                    <task> 
                      <statement>
                        <p>Using only the coefficients of the dividend, fill in the blanks in the top row below.</p>
                       
                      
                    <image xml:id="example-synthetic-division-coefficients" width="30%">
                      <latex-image>
                        \begin{tikzpicture}[x=1.1em, y=1.1em]
                          \draw[black!10] (-1,-1) rectangle (3,2);
                          \draw (0,2) -- (0,0) -- (3,0);
                          <!-- \node at (-0.5,1.5) {?}; -->
                          \node at (0.5,1.5) {?};
                          \node at (1.5,1.5) {?};
                          \node at (2.5,1.5) {7};
                          <!-- \node at (1.5,0.5) {?};
                          \node at (2.5,0.5) {?};
                          \node at (0.5,-0.5) {3};
                          \node at (1.5,-0.5) {2};
                          \node at (2.5,-0.5) {5}; -->
                        \end{tikzpicture}
                      </latex-image>
                      </image>
                    </statement>
                    </task>
                    <task> 
                      <statement>
                        <p>Find the zero of the divisor. Put this in front of the other coefficients. </p>
                        
                          <image xml:id="example-synthetic-missing-divisor" width="30%">
                            <latex-image>
                              \begin{tikzpicture}[x=1.1em, y=1.1em]
                                \draw[black!10] (-1,-1) rectangle (3,2);
                                \draw (0,2) -- (0,0) -- (3,0);
                                \node at (-0.5,1.5) {?};
                                \node at (0.5,1.5) {?};
                                \node at (1.5,1.5) {?};
                                \node at (2.5,1.5) {7};
                                <!-- \node at (1.5,0.5) {?};
                                \node at (2.5,0.5) {?};
                                \node at (0.5,-0.5) {3};
                                \node at (1.5,-0.5) {2};
                                \node at (2.5,-0.5) {5}; -->
                              \end{tikzpicture}
                            </latex-image>
                            </image>
                          </statement>
                    </task>
                    <task> 
                      <statement>
                        <p>How can we use the coefficients of the dividend and the zero of the divisor to produce the coefficients of the quotient in part b of Activity 4.3.8? </p>
                        <image xml:id="example-synthetic-final_step" width="30%">
                          <latex-image>
                            \begin{tikzpicture}[x=1.1em, y=1.1em]
                              \draw[black!10] (-1,-1) rectangle (3,2);
                              \draw (0,2) -- (0,0) -- (3,0);
                              \node at (-0.5,1.5) {?};
                              \node at (0.5,1.5) {?};
                              \node at (1.5,1.5) {?};
                              \node at (2.5,1.5) {7};
                              \node at (1.5,0.5) {?};
                              \node at (2.5,0.5) {?};
                              \node at (0.5,-0.5) {3};
                              \node at (1.5,-0.5) {2};
                              \node at (2.5,-0.5) {5};
                            \end{tikzpicture}
                          </latex-image>
                          </image>
                        </statement>
                    </task>
         <image xml:id="example-synthetic-division-coefficients" width="30%">
                      <latex-image>
                        \begin{tikzpicture}[x=1.1em, y=1.1em]
                          \draw[black!10] (-1,-1) rectangle (3,2);
                          \draw (0,2) -- (0,0) -- (3,0);
                          <!-- \node at (-0.5,1.5) {?}; -->
                          \node at (0.5,1.5) {?};
                          \node at (1.5,1.5) {?};
                          \node at (2.5,1.5) {7};
                          <!-- \node at (1.5,0.5) {?};
                          \node at (2.5,0.5) {?};
                          \node at (0.5,-0.5) {3};
                          \node at (1.5,-0.5) {2};
                          \node at (2.5,-0.5) {5}; -->
                        \end{tikzpicture}
                      </latex-image>
                      </image>
                    </statement>
                    </task>
                    <task> 
                      <statement>
                        <p>Find the zero of the divisor. Put this in front of the other coefficients. </p>
                        
                          <image xml:id="example-synthetic-missing-divisor" width="30%">
                            <latex-image>
                              \begin{tikzpicture}[x=1.1em, y=1.1em]
                                \draw[black!10] (-1,-1) rectangle (3,2);
                                \draw (0,2) -- (0,0) -- (3,0);
                                \node at (-0.5,1.5) {?};
                                \node at (0.5,1.5) {?};
                                \node at (1.5,1.5) {?};
                                \node at (2.5,1.5) {7};
                                <!-- \node at (1.5,0.5) {?};
                                \node at (2.5,0.5) {?};
                                \node at (0.5,-0.5) {3};
                                \node at (1.5,-0.5) {2};
                                \node at (2.5,-0.5) {5}; -->
                              \end{tikzpicture}
                            </latex-image>
                            </image>
                          </statement>
                    </task>
                    <task> 
                      <statement>
                        <p>How can we use the coefficients of the dividend and the zero of the divisor to produce the coefficients of the quotient in part b of Activity 4.3.8? </p>
                        <image xml:id="example-synthetic-final_step" width="30%">
                          <latex-image>
                            \begin{tikzpicture}[x=1.1em, y=1.1em]
                              \draw[black!10] (-1,-1) rectangle (3,2);
                              \draw (0,2) -- (0,0) -- (3,0);
                              \node at (-0.5,1.5) {?};
                              \node at (0.5,1.5) {?};
                              \node at (1.5,1.5) {?};
                              \node at (2.5,1.5) {7};
                              \node at (1.5,0.5) {?};
                              \node at (2.5,0.5) {?};
                              \node at (0.5,-0.5) {3};
                              \node at (1.5,-0.5) {2};
                              \node at (2.5,-0.5) {5};
                            \end{tikzpicture}
                          </latex-image>
                          </image>
                        </statement>
                    </task>
```

Comment (StevenClontz) [reply source/04-PR/03.ptx:294]: Result:

image

image

Comment (siwelwerd) [reply source/04-PR/03.ptx:294]: @sheppardw If you like @StevenClontz 's changes, there is a button right below his code that says "Commit suggestion" that you can hit to incorporate them.  And then when you are happy, mark this ready for review.

**Comment by siwelwerd** [comment]
@sheppardw Are you still working on this one?

**Comment by sheppardw** [comment]
How can I view the changes or does this last change have to be approved first before I can see it to mark it ready for review?
Thanks!!

**Comment by StevenClontz** [comment]
The red X by the "Test Build" reveals that we have an error: `critical: ID example-synthetic-division-coefficients already defined, line 305, column 77`. So we need to rename the second `xml:id="example-synthetic-division-coefficients"` to something else.

To do this (and preview the changes to make sure they are to your satisfaction), you can open this branch in a Codespace. If you have an existing Codespace, I'd change the branch like so:

image

image

**Comment by sheppardw** [comment]
Thank you! I am still not seeing the changes to the synthetic division coding when I go to the branch SheppardPR4Sec3LongDivision.
Once I click on the changes that you suggested, is it automatically included? Thanks!

**Comment by StevenClontz** [comment]
Try clicking the 🔃 icon next to the branch name in this picture:

image

**Comment by sheppardw** [comment]
Could it be because I accidentally put another section (edits for PR 4-3) on this branch? It will not go forward without dealing with that? I don't need that one. I submitted it correctly on another branch. Hope this makes sense... :-/

**Comment by StevenClontz** [comment]
Gotcha. Paste this into a terminal.

```
git fetch origin
git reset --hard origin/SheppardPR4Sec3LongDivision
```

**Comment by StevenClontz** [comment]
Oh shoot, I should have thought of that. Yes, it won't build if the XML is malformed so you wouldn't see the changes.

**Comment by siwelwerd** [comment]
I couldn't follow the logic of the synthetic division, so I removed it from this PR so that we can go ahead and mark this section as done.  We can consider a future PR that adds it back in with better scaffolding.

**Review by siwelwerd — APPROVED** [review]

**Pull request merged**